addons/account/static/src/js/account_payment_field.js

```diff
@@ -126,6 +127,26 @@ var ShowPaymentLineWidget = AbstractField.extend({
                 self.trigger_up('reload');
             });
     },
+    /**
+     * @private
+     * @override
+     * @param {MouseEvent} event
+     */
+    _onOutstandingCreditOpenJournalEntry: function (event) {
+        event.stopPropagation();
+        event.preventDefault();
+        var self = this;
+        var move_id = $(event.target).data('id');
+        if (move_id !== undefined && !isNaN(move_id)){
+            this._rpc({
+                    model: 'account.move',
+                    method: 'action_view_entry',
+                    args: [move_id],
+                }).then((action) => {
+                    self.do_action(action);
+                });
+            }
```

Comment:
plz indent this part

Reply:
Also, it should be very nice to reuse the same logique as the `view` button when the line is already reconciled. Why not use `_onOpenPayment` instead?